poofarmer/Images/mouse-hand-closed.png.import

```diff
@@ -0,0 +1,35 @@
+[remap]
```

Comment:
I can't leave a comment on an image, but I'm just wondering if we're able to get these hand cursors to be more centered or with less whitespace?

![image](https://user-images.githubusercontent.com/15854173/213964494-be92b9a4-710c-49e4-bb6e-2b5a8911eb79.png)


Or is it perhaps that we had to use these dimensions?